files/ohpc_install.yml

```diff
@@ -154,13 +167,7 @@
                                 compute_prefix='d050'
                                 compute_regex='d050[1-3]'
                                 kargs=' modprobe.blacklist=hibmc_drm'
-                                enable_opensm=False
-                                enable_ipoib=False
-                                enable_ifup=False
-                                enable_linux_ib=False
-                                enable_mellanox_ib=False
-                                mofed_install_opts=""
-                                enable_socket_direct=False
+                                mofed_install_opts="--distro rhel7.5alternate"
```

Comment:
this depends on the OS being installed (we're adding Suse), so it needs to come from somewhere else.

Reply:
Agreed.